src/app/core/components/modal-window/modal-window.component.scss

```diff
@@ -32,7 +32,7 @@
     cursor: pointer;
```

Comment:
The text inside the buttons is not aligned. To align the text vertically inside the button you should use `align-items: center;` instead of paddings.